lib/server.js

```diff
@@ -8,6 +8,10 @@
  * flushdb
  */
 exports.flushdb = flushdb = function (mockInstance, callback) {
+  for (var key in mockInstance.storage) {
```

Comment:
This could add a performance penalty if someone adds > 100000 keys ... 1 million keys takes 3 seconds on my hardware.